docs/test-content.md

```diff
@@ -38,14 +38,14 @@
 
 Then do *either* of the following:
 
-- Use [Bento4](#bento4) to split the file into an init segment and a fmp4 media segments:
+- Use [Bento4][bento4] to split the file into an init segment and a fmp4 media segments:
```

Comment:
Technically I don't think the bento4 links go anywhere.

Reply:
they will point at `[bento4]: https://www.bento4.com/documentation/`

Reply:
You're right, I was actually clicking the one on line 31

Reply:
oh, I guess I missed that one, let me update it too